client/src/components/Tickets/TicketModal.js

```diff
@@ -170,6 +185,14 @@ const TicketHistoryCard = ({
       </CardHeader>
       <Collapse isOpen={isOpen}>
         <CardBody>{content}</CardBody>
+        <CardBody
+          // eslint-disable-next-line react/jsx-props-no-spreading
+          {...(attachments = (attachments || []).filter(
+            a => !a[1].toString().startsWith('untitled (')
+          ))}
```

Comment:
Rather than reassigning the attachments here, can you create a new variable up above line 166 instead? That way we don't need to add any eslint disable lines

Reply:
yes, agree.  same comment from https://github.com/TACC/Core-Portal/pull/509#discussion_r716787307 which looks like it was inadvertently marked as resolved.